setup.py

```diff
@@ -18,7 +18,7 @@
                     'pint>=0.8',
                     'scipy>=1.0',
                     'traitlets>=4.3',
-                    'xarray>=0.10',
+                    'xarray>=0.11',
```

Comment:
Are you just updating the version number, or do we need features from the newest xarray?

Reply:
In this branch it was upgraded from 0.10 to 0.12 and then downgraded to 0.11. Wouldn't be surprised if the downgrade should have just been back to 0.10.

Reply:
yup, i meant to put that back to 0.10, i'll push it back